movie-browser/src/view/FeaturedMovie.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CardDetail from './CardDetail.js'
+
+// TODO: makes toggle for detail view work. have no idea why it's not working for this one
+// toggle: https://stackoverflow.com/questions/29913387/show-hide-components-in-reactjs
+class FeaturedMovie extends React.Component{
+  state = {isDetailShown: false}
+
+  handleClick = () => {
+    this.setState(preState => ({isDetailShown: !preState.isDetailShown}));
+    console.log(this.props.movie)
+  }
+
+  render() {
+    const { movie, isDetailShown } = this.props;
+    const { title, director, release_date } = movie
+    return (
+      <div className="card" onClick={() => this.handleClick()}>
```

Comment:
this should be `{this.handleClick}` . You already defined it on line 9

Reply:
Fixed. Now I (finally) understand what is going on there. I think I hadn't really understood what I was doing.